data/anaconda.conf

```diff
@@ -91,6 +91,9 @@ updates_repositories =
 #
 enabled_repositories_from_treeinfo = addon optional variant
 
+# Show experimental kernel option in the menu.
+show_gemini_kernel = False
```

Comment:
Since it is related only to the user interface, I would consider to use the `[User Interface]` section.